docs/querying/basics.md

```diff
@@ -21,6 +21,37 @@ Other programs can fetch the result of a PromQL expression via the [HTTP API](ap
 This document is a Prometheus basic language reference. For learning, it may be easier to
 start with a couple of [examples](examples.md).
 
+## Samples
+
+The value of a sample at a given timestamp returned by PromQL may be a float or
+a [native histogram](https://prometheus.io/docs/specs/native_histograms).
+
+Native histograms can be optionally scraped by Prometheus or received by
+Prometheus over remote-write or OTLP.
+
+Once native histograms have been ingested into the TSDB, both instant vectors
+and range vectors may now contain samples that aren't simple floating point
+numbers (float samples), but complete histograms (histogram samples). A vector
+may contain a mix of float samples and histogram samples. Note that the term
+“histogram sample” in the PromQL documentation always refers to a native
+histogram. Classic histograms are broken up into a number of series of float
+samples. From the perspective of PromQL, there are no “classic histogram
+samples”.
```

Comment:
Hmm, maybe we should have a separate `NOTE:` box that explains the "old" notion of classic histograms, maybe even quickly explaining what we mean by classic histograms. I could imagine that many readers of this aren't aware of that term.